features/git-town-append/on-parent-branch.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Appending a branch to a parent branch
```

Comment:
Auhh okay. I would only believe this test is needed then if we restricted branches to a single child. Since we don't, "its child" isn't really a concept in my mind. Branches can have many children, but we don't have one child we treat special.